pennylane_lightning/src/util/Util.hpp

```diff
@@ -31,6 +31,14 @@
 #include <type_traits>
 #include <vector>
 
+#if __has_include(<version>)
+#include <version>
+#endif
+
+#if __cpp_lib_math_constants >= 201907L
```

Comment:
Are these values compiler specific, or language version spec specific?
Should msvc/clang be affected by these values?

Reply:
Ah, I see. So once we get full C++20 support across the compilers we can get rid of these types of checks?